example/src/App.tsx

```diff
@@ -365,14 +363,16 @@ const App = () => {
           }}
         />
       </View>
-    </NavigationContainer>
+    </SafeAreaView>
   );
 };
 
 const styles = StyleSheet.create({
   container: {
-    paddingTop: 80,
-    flex: 1, // Enables flexbox column layout
+    height: Platform.select({
+      ios: Dimensions.get('window').height-44, // For top safe area
```

Comment:
operators should have spaces surrounding them, otherwise it's harder to read and it might read like it is the same expression with a dash
```suggestion
      ios: Dimensions.get('window').height - 44, // For top safe area
```